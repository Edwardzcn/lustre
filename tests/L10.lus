
function f<<const x : int>> () returns (y : int)
let
  y = x;
tel

function main(x : int) returns (y : int)
let
  --%MAIN
  y = f<<3>>();
tel


