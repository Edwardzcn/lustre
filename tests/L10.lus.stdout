node (x : int)
  returns (y)
  assumes ()
  shows ()
let
  y : int
    = 3
tel
--- Step 1 ---
x : int  = 1
y = 3
--- Step 2 ---
x : int  = 2
y = 3
--- Step 3 ---
x : int  = 3
y = 3
--- Step 4 ---
x : int  = (EOF)
